src/dialects/mariadb/query-generator.js

```diff
@@ -1,14 +1,57 @@
 'use strict';
 
+import { normalizeDataType } from '../abstract/data-types-utils';
+
 const { MySqlQueryGenerator } = require('../mysql/query-generator');
 const _ = require('lodash');
+const Utils = require('../../utils');
```

Comment:
tiny detail but prefer importing `joinSQLFragments` (using `import` too) instead of `Utils` in new code, we're slowly phasing our require for import